### ISSUE TITLE
Unable to resolve module ../SpriteAsync

### ISSUE THREAD

**Issue by pieterbeulque** [expo/expo-pixi#56]
I think this file doesn't work anymore in 1.0.0 because of the API deprecations.

https://github.com/expo/expo-pixi/blob/235f1a7df02767b564ae6230dc4315d75f669c48/lib/components/FilterImage.js#L8

**Comment by raskri** [comment]
Same for spineAsync.js -> './textureAsync' not found

**Comment by pieterbeulque** [comment]
@raskri - Indeed, that's also fixed in #57

**Comment by pieterbeulque** [comment]
@bunyaminmrcn - I'm afraid that has nothing to do with this issue

**Comment by mikelgarciaurbina** [comment]
Does someone know how fix this error?

**Comment by pieterbeulque** [comment]
@mikelgarciaurbina - A fix lives in #57 

I point my `package.json`-file to that branch of my fork and that works for me. It's not a permanent fix, of course, but it allows me to keep working.

```json
"expo-pixi": "git://github.com/pieterbeulque/expo-pixi.git#patch-1"
```

**Comment by mikelgarciaurbina** [comment]
Thanks @pieterbeulque I install the previous version (0.3.2) and is working.
